### ISSUE TITLE
upkeep: remove snapshop test

### ISSUE THREAD

**Issue by jacobvjk** [RMI-PACTA/r2dii.data#385]
snapshot tests are a hindrance to submitting updates of the r2dii.* suite to CRAN, because of interdependencies between the r2dii packages that may affect the snap results.

we want to remove them.

**Comment by jdhoffa** [comment]
I don't believe this is applicable in this case actually, since `r2dii.data` has no dependencies on other `r2dii` packages

**Comment by jacobvjk** [comment]
valid, closing
